http_examples.cpp

```diff
@@ -57,13 +57,16 @@ int main() {
 
             string name=pt.get<string>("firstName")+" "+pt.get<string>("lastName");
 
-            *response << "HTTP/1.1 200 OK\r\nContent-Length: " << name.length() << "\r\n\r\n" << name;
+            *response << "HTTP/1.1 200 OK\r\nContent-Length: " << name.length() << "\r\n" 
```

Comment:
I just searched about that. According to [RFC 2616](https://tools.ietf.org/html/rfc2616), It does not matter but there are some suggestions. I am not expert in this area. You can fix the orders.

[RFC 2616](https://tools.ietf.org/html/rfc2616) says:

> The order in which header fields with differing field names are
>    received is not significant. However, it is "good practice" to send
>    general-header fields first, followed by request-header or response-
>    header fields, and ending with the entity-header fields.